tenable/io/v3/vm/policies/api.py

```diff
@@ -0,0 +1,395 @@
+                    ...              'value': '2',
+                    ...              'operator': 'oper',
+                    ...              'property': '2'
+                    ...          }
+                    ...      ]
+                    ...  }],
+                    ...  'and': [{
+                    ...      'value': '3',
+                    ...      'operator': 'oper',
+                    ...      'property': 3
+                    ...  }]
+                    ... }
+                As the filters may change and sortable fields may change over
+                time, it's highly recommended that you look at the output of
+                the :py:meth:`tio.v3.definitions.vm.policies()`
+                endpoint to get more details.
+            sort (list[tuple], optional):
+                A list of dictionaries describing how to sort the data
+                that is to be returned.
+                Examples:
+                    >>> [('field_name_1', 'asc'), ('field_name_2', 'desc')]
+            limit (int, optional):
+                Number of objects to be returned in each request.
+                Default and max_limit is 200.
+            next (str, optional):
+                The pagination token to use when requesting the next page of
+                results. This token is presented in the previous response.
+            return_resp (bool, optional):
+                If set to true, will override the default behavior to return
+                an iterable and will instead return the results for the
+                specific page of data.
+            return_csv (bool, optional):
+                If set to true, it will return the CSV response or
+                iterable (based on return_resp flag). Iterator returns all
+                rows in text/csv format for each call with row headers.
+        Returns:
+            iterable:
+                The iterable that handles the pagination for the job.
+            requests.Response:
+                If ``return_json`` was set to ``True``, then a response
+                object is instead returned instead of an iterable.
+        Examples:
+            >>> tio.v3.vm.policies.search(
+            ...     filter=filters,
+            ...     fields=field, limit=2)
+        '''
+        iclass = SearchIterator
+
+        if kwargs.get('return_csv', False):
+            iclass = CSVChunkIterator
+
+        return super()._search(
+            iterator_cls=iclass,
+            sort_type=self._sort_type.default,
+            api_path=f'{self._path}/search',
+            resource='policies',
+            **kwargs
+        )
+
+    def templates(self) -> Dict:
+        '''
+        returns a dictionary of the scan policy templates using the
+        format of dict['name'] = 'UUID'.  This is useful for being able to
+        define scan policy templates w/o having to remember the UUID for each
+        individual one.
+        '''
+        # policies = dict()
```

Comment:
We can keep as of now, with V3 API integration we can enable that part since this branch will merge with [Task/in 581 v3 standardization](https://github.com/dhvanilshah-tenable/pyTenable/pull/104/files#)